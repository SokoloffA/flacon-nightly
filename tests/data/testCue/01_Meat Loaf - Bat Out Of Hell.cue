  REM DATE 1977
  PERFORMER "Meat Loaf"
    TITLE "Bat Out Of Hell"
FILE "Side 1.flac" WAVE
  TRACK 01 AUDIO
    TITLE "1.1 Bat Out Of Hell"
    INDEX 01 00:00:00
  TRACK 02  AUDIO
    TITLE "1.2 You Took The Words Right Out Of My Mouth (Hot Summer Night)"
    INDEX 01 09:52:00
  TRACK 03 AUDIO
    TITLE "1.3 Heaven Can Wait"
    INDEX 01 14:57:00
  TRACK 04 AUDIO
    TITLE "1.4 All Revved Up With No Place To Go"
    INDEX 01 19:39:00
FILE "Side 2.flac" WAVE
  TRACK 05 AUDIO
    TITLE "2.1 Two Out Of Three Ain't Bad"
    INDEX 01 00:00:00
  TRACK 06 AUDIO
    TITLE "2.2 Paradise By The Dashboard Light"
    INDEX 01 05:24:00
  TRACK 07 AUDIO
    TITLE "2.3 For Crying Out Loud"
    INDEX 01 13:49:00
  