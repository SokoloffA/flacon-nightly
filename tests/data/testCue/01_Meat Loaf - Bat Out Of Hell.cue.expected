[ DISC 01 / TRACK 01]
	FILE        = Side 1.flac
	INDEX CD 00 = 00:00:00
	INDEX CD 01 = 00:00:00
	TITLE       = 1.1 Bat Out Of Hell
	ALBUM       = Bat Out Of Hell
	PERFORMER   = Meat Loaf
	DATE        = 1977
	DISCID      =
	TRACKNUM    = 1
	TRACKCOUNT  = 7
	DISCNUM     = 1
	DISCCOUNT   = 1

[ DISC 01 / TRACK 02]
	FILE        = Side 1.flac
	INDEX CD 00 = 09:52:00
	INDEX CD 01 = 09:52:00
	TITLE       = 1.2 You Took The Words Right Out Of My Mouth (Hot Summer Night)
	ALBUM       = Bat Out Of Hell
	PERFORMER   = Meat Loaf
	DATE        = 1977
	DISCID      =
	TRACKNUM    = 2
	TRACKCOUNT  = 7
	DISCNUM     = 1
	DISCCOUNT   = 1


[ DISC 01 / TRACK 03]
	FILE        = Side 1.flac
	INDEX CD 00 = 14:57:00
	INDEX CD 01 = 14:57:00
	TITLE       = 1.3 Heaven Can Wait
	ALBUM       = Bat Out Of Hell
	PERFORMER   = Meat Loaf
	DATE        = 1977
	DISCID      =
	TRACKNUM    = 3
	TRACKCOUNT  = 7
	DISCNUM     = 1
	DISCCOUNT   = 1


[ DISC 01 / TRACK 04]
	FILE        = Side 1.flac
	INDEX CD 00 = 19:39:00
	INDEX CD 01 = 19:39:00
	TITLE       = 1.4 All Revved Up With No Place To Go
	ALBUM       = Bat Out Of Hell
	PERFORMER   = Meat Loaf
	DATE        = 1977
	DISCID      =
	TRACKNUM    = 4
	TRACKCOUNT  = 7
	DISCNUM     = 1
	DISCCOUNT   = 1


[ DISC 01 / TRACK 05]
	FILE        = Side 2.flac
	INDEX CD 00 = 00:00:00
	INDEX CD 01 = 00:00:00
	TITLE       = 2.1 Two Out Of Three Ain't Bad
	ALBUM       = Bat Out Of Hell
	PERFORMER   = Meat Loaf
	DATE        = 1977
	DISCID      =
	TRACKNUM    = 5
	TRACKCOUNT  = 7
	DISCNUM     = 1
	DISCCOUNT   = 1


[ DISC 01 / TRACK 06]
	FILE        = Side 2.flac
	INDEX CD 00 = 05:24:00
	INDEX CD 01 = 05:24:00
	TITLE       = 2.2 Paradise By The Dashboard Light
	ALBUM       = Bat Out Of Hell
	PERFORMER   = Meat Loaf
	DATE        = 1977
	DISCID      =
	TRACKNUM    = 6
	TRACKCOUNT  = 7
	DISCNUM     = 1
	DISCCOUNT   = 1


[ DISC 01 / TRACK 07]
	FILE        = Side 2.flac
	INDEX CD 00 = 13:49:00
	INDEX CD 01 = 13:49:00
	TITLE       = 2.3 For Crying Out Loud
	ALBUM       = Bat Out Of Hell
	PERFORMER   = Meat Loaf
	DATE        = 1977
	DISCID      =
	TRACKNUM    = 7
	TRACKCOUNT  = 7
	DISCNUM     = 1
	DISCCOUNT   = 1
